wallet/test/wallet.go

```diff
@@ -35,6 +35,7 @@ type Setup struct {
 	Backend         wallet.Backend // backend implementation
 	Wallet          wallet.Wallet  // the wallet instance used for testing
 	AddressInWallet wallet.Address // an address of an account in the test wallet
+	ZeroAddress     wallet.Address // an address that is less or equal to any other address
```

Comment:
Why less **or** equal? An address should never be equal to anything but itself.

Reply:
A zero address can be equal to another zero address.

Reply:
Ok, so there can be multiple zero addresses.